doc/source/rllib/core-concepts.rst

```diff
@@ -42,39 +98,237 @@ define a trainable policy with a functional-style API, for example:
       name="MyTFPolicy",
       loss_fn=policy_gradient_loss)
 
+
+Policy Evaluation
+-----------------
+
+Given an environment and policy, policy evaluation produces `batches <https://github.com/ray-project/ray/blob/master/rllib/policy/sample_batch.py>`__ of experiences. This is your classic "environment interaction loop". Efficient policy evaluation can be burdensome to get right, especially when leveraging vectorization, RNNs, or when operating in a multi-agent environment. RLlib provides a `RolloutWorker <https://github.com/ray-project/ray/blob/master/rllib/evaluation/rollout_worker.py>`__ class that manages all of this, and this class is used in most RLlib algorithms.
+
+You can use rollout workers standalone to produce batches of experiences. This can be done by calling ``worker.sample()`` on a worker instance, or ``worker.sample.remote()`` in parallel on worker instances created as Ray actors (see `WorkerSet <https://github.com/ray-project/ray/blob/master/rllib/evaluation/worker_set.py>`__).
+
+Here is an example of creating a set of rollout workers and using them gather experiences in parallel. The trajectories are concatenated, the policy learns on the trajectory batch, and then we broadcast the policy weights to the workers for the next round of rollouts:
+
+.. code-block:: python
+
+    # Setup policy and rollout workers
+    env = gym.make("CartPole-v0")
+    policy = CustomPolicy(env.observation_space, env.action_space, {})
+    workers = WorkerSet(
+        policy_class=CustomPolicy,
+        env_creator=lambda c: gym.make("CartPole-v0"),
+        num_workers=10)
+
+    while True:
+        # Gather a batch of samples
+        T1 = SampleBatch.concat_samples(
+            ray.get([w.sample.remote() for w in workers.remote_workers()]))
+
+        # Improve the policy using the T1 batch
+        policy.learn_on_batch(T1)
+
+        # Broadcast weights to the policy evaluation workers
+        weights = ray.put({"default_policy": policy.get_weights()})
```

Comment:
Maybe add here: Put the "parameter server's" (local worker's) weights once into the object store, so we don't have to do this n times (n=num remote workers).

Reply:
done